workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs

```diff
@@ -27,6 +27,8 @@ internal class RequiredFieldInjector
             = "[Require] attribute found on member that is not Injectable. This member will be ignored.";
         private const string MalformedInjectable
             = "Injectable found without required attributes, this is invalid.";
+        private const string RequirableFieldDoesNotInheritRequirableBase
+            = "[Require] field element does not inherit RequirableBase. This is most likely a bug in the Unity GDK.";
```

Comment:
Couldn't we just replace the Injectable interface with the RequirableBase class? Sounds like that would make more sense. Not sure if we should do it now or just put it in a log of "cleanup refactors to do" though because it's literally everywhere though

Reply:
Either way I don't like the error message. Firstly we are "SpatialOS GDK for Unity" now, but also maybe merge with MalformedInjectable? They are both for messed up classes that shouldn't happen at all

Reply:
1st comment can be part of UTY-1144 - please add to the acceptance criteria

Reply:
I don't disagree with the idea of getting rid of the `IInjectable` interface but let's do this in another PR. This PR is kinda bloated already. :)